jobs/dap-collector-ppa-dev/dap_collector_ppa_dev/main.py

```diff
@@ -126,15 +127,16 @@ async def collect_once(task, timestamp, duration, hpke_private_key, auth_token):
                     if ad is not None:
                         cnt = {}
                         cnt["collection_time"] = collection_time
-                        cnt["placement_id"] = ad["debug"]["placementId"]
-                        cnt["ad_id"] = ad["debug"]["adId"]
-                        cnt["conversion_key"] = ad["debug"]["conversionKey"]
+                        cnt["placement_id"] = ad["advertiserInfo"]["placementId"]
```

Comment:
I adjusted the shape of the data in ads.json so that the job isn't reading a bunch of debug fields. The config is only used by this job and many of the fields exist to help enable similar aggregations as external reports.